service/step_defs_test.go

```diff
@@ -1165,7 +1170,6 @@ func (f *feature) iCallNodeGetInfo() error {
 	f.nodeGetInfoResponse, f.err = f.service.NodeGetInfo(context.Background(), req)
 	if f.err != nil {
 		log.Printf("NodeGetInfo call failed: %s\n", f.err.Error())
-		return f.err
```

Comment:
any reason why errors being ignored here ?

Reply:
brought it back.